### PR TITLE
doc: Fix PSBT howto and example parameters

### PR DESCRIPTION
**Pull request by priscoan**

### CODE REVIEW

**Comment by achow101** [comment]
ACK f764fa661963df434847e217cadc3f07e2fba993

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -3938,7 +3938,7 @@ UniValue walletcreatefundedpsbt(const JSONRPCRequest& request)
 
     if (request.fHelp || request.params.size() < 2 || request.params.size() > 5)
         throw std::runtime_error(
-                            "walletcreatefundedpsbt [{\"txid\":\"id\",\"vout\":n},...] [{\"address\":amount},{\"data\":\"hex\"},...] ( locktime ) ( replaceable ) ( options bip32derivs )\n"
+                            "walletcreatefundedpsbt [{\"txid\":\"id\",\"vout\":n},...] [{\"address\":amount},{\"data\":\"hex\"},...] ( locktime ) ( options bip32derivs )\n"
```
Comment (MarcoFalke) [inline src/wallet/rpcwallet.cpp]: This is already fixed in master. Could remove this commit and avoid the merge conflict?

Comment (MarcoFalke) [reply src/wallet/rpcwallet.cpp]: Done with a force push (Hope you don't mind)

**Comment by fanquake** [comment]
@achow101 Did you want to re-ack here

**Comment by achow101** [comment]
re-ACK 78542a3f36381fe96f60708834c8378042e3bab6

**Comment by fanquake** [comment]
utACK 78542a3

**Pull request merged**